homework_lection2_sergiy_baranov/main.py

```diff
@@ -0,0 +1,128 @@
+import json
+from pydantic import BaseModel
+from abc import ABC
+
+# Клас "Книга"
+class BookModel(BaseModel):
+    title: str
+    author: str
+    year_published: int
+
+    def __str__(self):
+        return f"Title: {self.title}, Author: {self.author}, Year Published: {self.year_published}"
+
+    def to_dict(self):
```

Comment:
Good that you did it, but pydantic v2 has method model_dump() which do the same with additional features.